src/components/FlowRunsBarChart.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <div ref="chart" class="flow-runs-bar-chart" :class="classes.root" :style="styles.root" @mouseleave="close">
     <template v-for="(flowRun, index) in barFlowRuns" :key="getKey(flowRun, index)">
-      <p-pop-over class="flow-runs-bar-chart__bar-container" :to="chart" :placement="placement" :group="group" auto-close>
```

Comment:
`:to` is the teleport target and the default is the body so then this change has the popover content teleporting out of the table thus solving the overflow: hidden issue. 

tho i wonder what use-case `:to="chart"` might be for 🤔